src/main/java/nextstep/courses/domain/LectureStatus.java

```diff
@@ -1,5 +1,8 @@
 package nextstep.courses.domain;
 
 public enum LectureStatus {
-    PREPARING, RECRUITING, COMPLETED
+    PREPARING, RECRUITING, COMPLETED;
+    public boolean isRecruiting() {
+        return this.equals(RECRUITING);
```

Comment:
```suggestion
        return this == RECRUITING;
```
enum은 jvm 상에 유일한 하나의 인스턴스를 가지기 때문에 위와 같이 구현 가능함